pubsublite/rpc_util.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+package pubsublite
```

Comment:
nit: Try to avoid filenames like util. The contents of this file seems related to retrying, so maybe `retry.go`?

Reply:
In future PRs, it will contain other low-level rpc utils, e.g. attaching additional headers, common ClientOptions, etc.

I renamed it to rpc.go - I don't know whether that's better or worse... :)